### PR TITLE
AET auth db updates

### PR DESCRIPTION
**Pull request by vbudhram**
@6a68 This is just a draft to get tests etc passing, lmk what you think.

### CODE REVIEW

**Comment by 6a68** [comment]
LGTM so far, thanks @vbudhram!

**Comment by 6a68** [comment]
I've merged this commit into my branch for #5689, I'll see if I can get CI tests to pass over there.

File: packages/fxa-auth-db-mysql/db-server/index.js [hunk 1]
```diff
@@ -126,6 +126,10 @@ function createServer(db) {
   );
   api.post('/account/:id/locale', withIdAndBody(db.updateLocale));
   api.get('/account/:id/sessions', withIdAndBody(db.sessions));
+  api.put(
```
Comment (jodyheavener) [inline packages/fxa-auth-db-mysql/db-server/index.js:129]: I think this should be a PATCH instead of PUT given we're only updating a single row, no?

**Comment by vbudhram** [comment]
Closing this in favor of https://github.com/mozilla/fxa/pull/5689

File: packages/fxa-auth-db-mysql/db-server/index.js (hunk 1 above)
Comment (vbudhram) [reply packages/fxa-auth-db-mysql/db-server/index.js:129]: I have to double check if restify supports this, I think technically either is fine.

**Pull request closed without merging**